src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardConsumerTest.java

```diff
@@ -97,7 +97,7 @@
     private final boolean skipCheckpointValidationValue = false;
     private static final InitialPositionInStreamExtended INITIAL_POSITION_LATEST =
             InitialPositionInStreamExtended.newInitialPosition(InitialPositionInStream.LATEST);
-    private static final ShardSyncer shardSyncer = new ShardSyncer(new KinesisLeaseCleanupValidator());
+    private static final KinesisShardSyncer shardSyncer = new KinesisShardSyncer(new KinesisLeaseCleanupValidator());
```

Comment:
Let's code to interfaces and not classes.

Reply:
KCL tests are specific to Kinesis behavior, hence all tests use KinesisShardSyncer (implementation of ShardSyncer). We have separate tests (in the adapter package) for the Streams specific implementation of ShardSyncer.